frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -23,16 +36,21 @@
         </b-button>
         <br />
         <b-button
-          class="p-2 mt-3"
+          class="p-2 mt-3 test-delete-link"
           size="sm"
-          variant="outline-danger"
           @click="deleteLink()"
           :title="$t('delete')"
         >
           <b-icon icon="trash"></b-icon>
         </b-button>
       </b-col>
     </b-row>
+    <b-modal :id="'modalPopover-' + id" title="QR-Code" ok-only hide-header-close>
+      <div class="text-center">
+        <figure-qr-code :text="link" />
+        <p>{{ link }}</p>
```

Comment:
so the link is not part of the image I guess?

Reply:
nein in der form ist der link nicht teil des bildes. das wäre etwas umständlicher aber möglich .. würde ich aber empfehlen in einem folge ticket zu realisieren.